enterprise/app/services/sla/evaluate_applied_sla_service.rb

```diff
@@ -44,9 +44,16 @@ def check_next_response_time_threshold(applied_sla, conversation, sla_policy)
     return if conversation.waiting_since.blank?
 
     threshold = conversation.waiting_since.to_i + sla_policy.next_response_time_threshold.to_i
```

Comment:
We could extract this logic to separate method since it used in multiple place.

```
def calculate_threshold(base_time, offset)
    base_time.to_i + offset.to_i
end
```